client/signup/steps/woocommerce-install/select-site/index.tsx

```diff
@@ -0,0 +1,64 @@
+import { isBusinessPlan } from '@automattic/calypso-products';
+import { Card } from '@automattic/components';
+import { useI18n } from '@wordpress/react-i18n';
+import { ReactElement, useState, useEffect } from 'react';
+import { useDispatch } from 'react-redux';
+import SiteSelector from 'calypso/components/site-selector';
+import StepWrapper from 'calypso/signup/step-wrapper';
+import { submitSignupStep } from 'calypso/state/signup/progress/actions';
+import type { WooCommerceInstallProps } from '../';
+
+import './style.scss';
+
+interface Site {
+	capabilities: {
+		manage_options: boolean;
+	};
+	jetpack: boolean;
+	plan: {
+		product_slug: string;
+	};
+}
+
+export default function SelectSite( props: WooCommerceInstallProps ): ReactElement | null {
+	const { goToStep } = props;
+	const { __ } = useI18n();
+
+	// todo: we should allow any (non jetpack, non p2) site to show up here,
+	// and push people through the checkout step as needed
+	const filterSites = ( site: Site ) => {
+		return (
+			isBusinessPlan( site.plan.product_slug ) && site.capabilities.manage_options && ! site.jetpack
```

Comment:
Could/should we remove the site plan check and handle it in the next step? We could redirect the customers to the checkout page easily. Also, is that currently handled by the landing page?

Reply:
Could/Should - probably, I coinflipped it because I wasn't sure what the best option was.